server/controllers/system/add-department.php

```diff
@@ -29,20 +28,15 @@ class AddDepartmentController extends Controller {
     public function validations() {
         return [
             'permission' => 'staff_3',
-            'requestData' => [
-                'name' => [
-                    'validation' => DataValidator::alnum(),
-                    'error' => ERRORS::INVALID_NAME
-                ]
-            ]
+            'requestData' => []
         ];
     }
 
     public function handler() {
         $name = Controller::request('name');
```

Comment:
Also, try to use `htmlentities` function, 
```
htmlentities('<b>new department</b>') === '&lt;b&gt;new department&lt;/b&gt'; // true
```
then in the test you can check if the value is `&lt;b&gt;new department&lt;/b&gt`